pygfx/helpers/_lights.py

```diff
@@ -149,10 +182,28 @@ def _update(self):
 
 
 class SpotLightHelper(Line):
-    """Helper class to visualize a spot light.
-    The helper object should be a parent of the light object.
+    """Light source indicator for spot lights.
+
+    Helper class to visualize a spot light. The helper object should be a child
+    of the light object.
+
+    Parameters
+    ----------
+    color : Color
+        The color of the arrows. If None, match the color of the light.
+
+    Examples
+    --------
+    .. minigallery:: pygfx.SpotLightHelper
+
     """
 
+    # @almarklein: What is the expected shape of this indicator? Also, could we
+    # combine its code path with that for the spot light, but make the Mesh a
+    # cone that points into the opposite direction of the light source (ie. in
+    # such a way that the bottom of the cone has the light coming out, like a
+    # lamp)
```

Comment:
They look like this:
<img width="566" alt="image" src="https://user-images.githubusercontent.com/3015475/212285624-8275966a-d9d6-4af8-a216-846e034498c3.png">

The idea of the helper to see quite precisely where it's pointed at, which would not be as clear with the "lamp". It could be worth improving the helper, but it'd require more discussion, so it'd be more suited in a separate issue/PR.

Reply:
That makes sense. In this case, lets take it up again once there is a specific need/use-case for an improvement.